docs/migration-pulp-installer-to-multi-process.md

```diff
@@ -0,0 +1,286 @@
+```
+sudo chown -R $USER:$(id -gn) settings
+podman unshare chown -R 700:700 settings
+```
+If running podman as root, or docker:
+```
+sudo chown -R 700:700 settings
+```
+
+### Configure Pulp to talk to the database (Fedora 36 and Ubuntu 22.04 only)
+
+If running podman, modify `settings/settings.py` so that "DATABASE" includes:
+```
+'HOST': 'host.containers.internal'
+```
+An example of what the line will look like:
+```
+DATABASES = {'default': {'HOST': 'host.containers.internal', 'ENGINE': 'django.db.backends.postgresql', 'NAME': 'pulp', 'USER': 'pulp', 'PASSWORD': 'pulp'}}
+```
+
+If running docker, modify `settings/settings.py` so that "DATABASE" includes:
+```
+'HOST': 'host.docker.internal'
+```
+An example of what the line will look like:
+```
+DATABASES = {'default': {'HOST': 'host.docker.internal', 'ENGINE': 'django.db.backends.postgresql', 'NAME': 'pulp', 'USER': 'pulp', 'PASSWORD': 'pulp'}}
+```
+
+### Manage the PostgreSQL data directory (EL, Fedora prior to 36, and Debian 11 only)
+
+1st, move the directory:
+
+If on EL, or Fedora prior to 36
+```
+sudo mv /var/lib/pgsql pgsql
+```
+If on Debian 11:
+```
+sudo mv /var/lib/postgresql/13 pgsql
+sudo mv pgsql/main pgsql/data
+```
+If on EL7 or EL8:
+```
+sudo mv /var/lib/pgsql/data /var/lib/pgsql/data_old
+```
+
+Next, if on Debian 11, move the config files:
+```
+sudo mv /etc/postgresql/13/main/*.conf /etc/postgresql/13/main/conf.d/ pgsql/data/
+```
+
+Next change ownership of the directory:
+
+If running rootless podman:
+```
+sudo chown -R $USER:$(id -gn) pgsql
+podman unshare chown -R 26:26 pgsql
+```
+If running podman as root, or docker:
+```
+sudo chown -R 26:26 pgsql
+```
+
+### Configure Postgres to be compatible with the EL8-based container (Debian 11 only)
+
+Backup `pgsql/data/postgresql.conf` before you modify it:
+```
+sudo mv pgsql/data/postgresql.conf pgsql/data/postgresql.conf.old
+```
+
+Next, comment out the following lines in `pgsql/data/postgresql.conf`:
+```
+data_directory
+hba_file
+ident_file
+external_pid_file
+unix_socket_directories
+ssl
+ssl_cert_file
+ssl_key_file
+cluster_name
+stats_temp_directory
+```
+
+### Create an empty containers directory
+
+We do not bother to move `/var/lib/containers` because it is only ever used for temporary files by pulp_ansible, and may be used on the host for other purposes (like running podman):
+
+```
+mkdir containers
+```
+
+### Restore the database (EL7 or EL8 only)
+
+Run the container with the normal [command](multi-process-images#starting-the-container), but with `-it` instead of `-detach`, and with `/bin/bash` as the specified command. 
+```
+podman run -it              --publish 8080:80              --name pulp              --volume "$(pwd)/settings":/etc/pulp:Z              --volume "$(pwd)/pulp_storage":/var/lib/pulp:Z              --volume "$(pwd)/pgsql":/var/lib/pgsql:Z              --volume "$(pwd)/containers":/var/lib/containers:Z              --device /dev/fuse              pulp/pulp /bin/bash
```

Comment:
This makes pulp available on port 8080. We should try to preserve the original port.